pyproject.toml

```diff
@@ -10,23 +10,23 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "3.9.5"
-django = "3.2.*"
+django = "3.*"
 django-filter = "2.*"
 djangorestframework = "3.*"
 djangorestframework-simplejwt = "4.*"
 pyjwt = "1.7.1"
 django-fsm = "2.*"
 requests = "2.*"
-channels = "2.*"
-channels_redis = {version = "3.0.*", extras=["cryptography"]}
-celery = {version = "4.4.*", extras=["redis"]}
+channels = "3.*"
```

Comment:
The change of channels 3, was to support the Django Async Changes. Does this impact anything? Have you thouroughly tested this?

Reply:
What I tested:
* unit tests pass
* websocket returns 101 and respond to query
* server spins up with daphne and asgi configuration